pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/globalcomputeforwardingrule/_generated_object_globalcomputeforwardingrule.golden.yaml

```diff
@@ -48,6 +32,5 @@ status:
     reason: UpToDate
     status: "True"
     type: Ready
-  labelFingerprint: abcdef0123A=
```

Comment:
Looks like we need to surface this field in status, because it's a beta resource.